scripts/validate_my_definition.py

```diff
@@ -43,7 +43,7 @@ def validate_definition(definition_file):
     options = parser.parse_args()
     r = requests.get(OPENAPI_SCHEMA_URL)
     schema = r.json()
-    files = glob.glob(f"{options.dir}/*.yaml")
+    files = glob.glob(f"{options.dir}/*.v*.yaml")
```

Comment:
maybe `"{options.dir}/*.v[1-9]*.yaml"`?

Reply:
[fnmatch](https://docs.python.org/3.7/library/fnmatch.html#module-fnmatch) looks like it supports ranges, but not the regex repetition or real optional characters, but this should be at least more accurate.

Reply:
glob.glob calls fnmatch, so the range should work in the argument here too.

Reply:
Updated.
```python
>>> glob.glob("*.v[1-9]*.yaml")
['remediations.v1.yaml', 'assets_query.v1.yaml']
```